spec/controllers/linked_data_terms_controller_spec.rb

```diff
@@ -344,6 +364,7 @@
             get :show, params: { id: 'c_9513', vocab: 'AGROVOC', format: 'jsonld' }
             expect(response).to be_successful
             expect(response.content_type).to eq 'application/ld+json'
+            expect(JSON.parse(response.body).keys).to match_array ["@context", "@graph"]
```

Comment:
Confirms that no only is the header set correctly but the response actually contains json-ld.  There was an error that got missed where the header was set correctly but the response had the original json format.  This check catches that error.